### PR TITLE
[it_IT] Translate CONTRIBUTING: Making Feature/Enhancement Proposals

### PR DESCRIPTION
**Pull request by FabrizioPe**

### CODE REVIEW

**Review by riki95 — APPROVED** [review]
I'll approve this but @FabrizioPe , as already mentioned in another PR, it makes no sense to do 1 PR a day with just 1 file changed.
Please do PR less frequently and with major changed, like a whole section translated.
It helps reviewing it with less confusion in my opinion. Thanks for your work.

**Comment by FabrizioPe** [comment]
Hi @riki95, sorry for the inconvenience, this workflow is just more convenient for our team as we can review each other's little pieces daily, but I understand it may be less convenient for other reviewers.
We'll discuss the issue and try to make the PR workflow more manageable.
Thanks for the feedback :)

File: docs/locale/it_IT/source/CONTRIBUTING.rst [hunk 1]
```diff
@@ -91,28 +91,25 @@ I maintainer del progetto, di volta in volta, prenderanno in considerazione l'ag
...
-Hacerlo y, si es posible, cómo podría implementarse. También ayudaría a argumentar por qué se debe agregar la función, como identificar casos de uso específicos para los cuales se necesita la función y un caso de cuál sería el beneficio si se implementara la función. Una vez que se crea el problema de la JIRA, y el "un paginador" ya sea adjunto, incluido en el campo de descripción, o un enlace a un documento de acceso público se agrega a la descripción, envíe un correo electrónico de presentación a fabric@lists.hyperledger.org lista de correo que vincula el tema de la JIRA y solicita comentarios.
+È possibile implementare e revisionare piccoli miglioramenti attraverso il normale `flusso di lavoro su GitHub <https://docs.github.com/en/get-started/quickstart/github-flow>`__ ma per cambiamenti più sostanziali Fabric segue la procedura RFC (*request for comments*, richiesta di commenti).
```
Comment (FrancescoUrbano) [inline docs/locale/it_IT/source/CONTRIBUTING.rst]: "processo" è diverso da "procedura", suggerirei di rendere con "il processo" qui e sotto

Comment (FrancescoUrbano) [inline docs/locale/it_IT/source/CONTRIBUTING.rst]: suggerisco di rendere come l'originale inglese: "flusso di lavoro delle pull request su GitHub"

**Review by FrancescoUrbano — CHANGES_REQUESTED** [review]

File: docs/locale/it_IT/source/CONTRIBUTING.rst (hunk 1 above)
Comment (FabrizioPe) [reply docs/locale/it_IT/source/CONTRIBUTING.rst]: ok, fatto

Comment (FabrizioPe) [reply docs/locale/it_IT/source/CONTRIBUTING.rst]: ok fatto

**Review by FrancescoUrbano — APPROVED** [review]
LGTM

**Pull request merged**